### PR TITLE
use same update object to display progress

### PR DESCRIPTION
**Pull request by colombod**
This fixes the issue of progress update on another line when importing packages

### CODE REVIEW

File: samples/vs-code/covid-19.dotnet-interactive [hunk 1]
```diff
@@ -118,7 +118,7 @@ notebookScope.plot = function (plotTarget) {
                         name: "Recovered",
                         type: "scattergeo",
                         mode: "markers",
-                        geo: "geo1",
```
Comment (jonsequitur) [inline samples/vs-code/covid-19.dotnet-interactive:121]: Do we want to make these same changes in the equivalent `.ipynb`?

**Comment by jonsequitur** [comment]
Maybe the VS Code notebooks should go under `/samples/notebooks`, alongside the Jupyter examples? They're still notebooks.

File: samples/vs-code/covid-19.dotnet-interactive (hunk 1 above)
Comment (colombod) [reply samples/vs-code/covid-19.dotnet-interactive:121]: let me do that

Comment (colombod) [reply samples/vs-code/covid-19.dotnet-interactive:121]: done

**Comment by colombod** [comment]
> Maybe the VS Code notebooks should go under `/samples/notebooks`, alongside the Jupyter examples? They're still notebooks.

they will be copied / mounted to the my binder image, I put those files in their own folder as I am not even sure we should have the ipynb files in a folder we open with vscode

**Pull request merged**